app/Http/Controllers/SettingsController.php

```diff
@@ -332,6 +332,7 @@ public function postSettings(Request $request) : RedirectResponse
         $setting->email_format = $request->input('email_format');
         $setting->username_format = $request->input('username_format');
         $setting->require_accept_signature = $request->input('require_accept_signature');
+        $setting->checkin_notification_user = $request->input('checkin_notification_user');
```

Comment:
We should always make sure to add a sane default for these, otherwise folks on Postgres have a bad time. (See the line right after this for an example.)

Reply:
I updates the Controller and migration to have 1 as the default value.